src/include/platform/NetworkCommissioning.h

```diff
@@ -276,6 +284,72 @@ class WiFiDriver : public Internal::WirelessDriver
      */
     virtual void ScanNetworks(ByteSpan ssid, ScanCallback * callback) = 0;
 
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFI_PDC
+    virtual bool SupportsPerDeviceCredentials() { return false; };
+
+    /**
+     * @brief Adds or updates a WiFi network with Per-Device Credentials on the device.
+     *
+     * @param ssid                          The SSID of the network to be added / updated.
+     * @param networkIdentity               The Network Identity of the network, in compact-pdc-identity format.
+     * @param clientIdentityNetworkIndex    If present, the index of the existing network configuration of which the Client
+     *                                      Identity is to be re-used. Otherwise a new Client Identity shall be generated.
+     * @param outStatus                     The application-level status code (Status::kSuccess on success).
+     * @param outDebugText                  A debug text buffer that may be populated by the driver when a Status is returned.
```

Comment:
This method now no longer ever returns a Status.  The comments need to be fixed.